### PR TITLE
Check the arguments on JavaScript side

### PR DESCRIPTION
**Pull request by minggangw**
Now we validated the argrments, which are passed from JavaScript, in C++ addon.
This escalated the cost, and as the binding methoths will be called by our
JS interfaces only, it's better to do the verification in JavaScript
instead.

Fix issue #12

### CODE REVIEW

**Comment by minggangw** [comment]
@kenny-y PTAL, thanks!

File: index.js [hunk 1]
```diff
@@ -31,14 +31,16 @@ let rcl = {
   _nodes: [],
 
   createNode(nodeName, namespace = '') {
-    let handle = rclnodejs.createNode(nodeName, namespace);
+    if (typeof (nodeName) !== 'string' || typeof (namespace) !== 'string') {
+      throw new Error('Wrong type of argments.');
```
Comment (kenny-y) [inline index.js]: Consider using TypeError: "The TypeError object represents an error when a value is not of the expected type" from:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypeError

Comment (kenny-y) [reply index.js]: Ref: https://github.com/RobotWebTools/rclnodejs/blob/develop/rosidl_gen/templates/message.dot#L117

**Review by kenny-y — APPROVED** [review]
Apart from the TypeError change request, the rest looks good. I'll make the check mark green in advance.

File: index.js (hunk 1 above)
Comment (minggangw) [reply index.js]: Yes, TypeError should be used, will correct and update the pr, thanks!

**Pull request merged**